locale/en/LC_MESSAGES/messages.po

```diff
@@ -2787,7 +2787,7 @@ msgid "Player Elo [Pts]"
 msgstr ""
 
 #, python-format
-msgid "Player [%(last_name)s %(first_name)s] is forfeit for the next round in tournament [%(tournament_uniq_id)s]."
+msgid "Player [%(last_name)s %(first_name)s] has a bye for the next round in tournament [%(tournament_uniq_id)s]."
```

Comment:
I'd rather have “is absent”.
A bye is more general and could be construed as “has a HPB”, or “has a FPB”.
```suggestion
msgid "Player [%(last_name)s %(first_name)s] is absent for the next round in tournament [%(tournament_uniq_id)s]."
```